lib/actions/term-groups.js

```diff
@@ -12,14 +12,16 @@ import getRootGroups from '../selectors';
 import {setActiveSession, ptyExitSession, userExitSession} from './sessions';
 
 function requestSplit(direction) {
-  return () => (dispatch, getState) => {
+  return (options) => (dispatch, getState) => {
```

Comment:
Please remove the parenthesis around `options` – it's not needed for a single function argument